.github/workflows/add-pr-to-project.yml

```
@@ -2,19 +2,28 @@ name: PR을 프로젝트에 자동 할당합니다.
+          sudo apt-get update
+          sudo apt-get install -y gh
+          echo "${{ secrets.JIYOUNG_PAT }}" | gh auth login --with-token
+
+      - name: Add PR to Project
+        env:
+          PR_URL: ${{ github.event.pull_request.html_url }}
+        run: |
+          set -e
+          echo "Adding PR to Project..."
+          gh project item-add 4 --owner devlink-community --url "$PR_URL"
+          echo "Done"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)